src/content/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration.mdx

```diff
@@ -0,0 +1,418 @@
+Be sure to insert your `<INSTANCE_ID>`:
+
+```yml lineHighlight=12-16
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: node
+
+    # Target setup 
+    static_configs:
+    - targets: ['localhost:9100']
+      labels:
+        instanceid: <INSTANCE_ID> 
+
+remote_write:
+  - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=NodeExporter
+    authorization:
+      credentials: YOUR_LICENSE_KEY
+```
+
+    </Collapser>
+    <Collapser
+        id="azure-static"
+        title="Azure"
+    >
+Be sure to insert your `<MACHINE_ID>`:
+
+```yml lineHighlight=12-16
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: node
+
+    # Target setup 
+    static_configs:
+    - targets: ['localhost:9100']
+      labels:
+        instanceid: <MACHINE_ID>
+
+remote_write:
+  - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=NodeExporter
+    authorization:
+      credentials: YOUR_LICENSE_KEY
+```
+    </Collapser>
+    <Collapser
+        id="gcp-static"
+        title="GCP"
+    >
+Be sure to insert your `<INSTANCE_ID>`:
+
+```yml lineHighlight=12-16
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: node
+
+    # Target setup 
+    static_configs:
+    - targets: ['localhost:9100']
+      labels:
+        instanceid: <INSTANCE_ID>
+
+remote_write:
+  - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=NodeExporter
+    authorization:
+      credentials: YOUR_LICENSE_KEY
+```    
+    </Collapser>
+</CollapserGroup>
+
+#### Dynamic targets [#dynamic-targets]
+Instead of configuring static targets, you can configure service discovery. 
```

Comment:
Should I know what this is? Why would I choose this instead?

Reply:
@homelessbirds I think users of this feature will be familiar with the pros and cons of static versus dynamic targets.